### DISCUSSION TITLE
How can I access which containers uses which overlays without using docker CLI.

### DISCUSSION THREAD

**Discussion by tchaton** [moby/moby#47913] [General]
I would like to be able by checking the files directly.

**Comment by polarathene** [comment]
If you want to inspect the image content, a TUI like `dive` is good for that. At run-time Docker Desktop GUI also works well for containers.

If you're not interested in image content or container run-time state, but files you're wanting to persist, use a bind mount volume and you can inspect content locally through your file browser?
